### DISCUSSION TITLE
Accordion : flash when i choose an another topic on Firefox / Edge and Safari

### DISCUSSION THREAD

**Discussion by geoffreywesse** [twbs/bootstrap#32078] [Show & Tell]
Hello !

Tested on Windows 10

Works well on :
Chrome

But not on : 
Firefox
Edge
Safari

Reduced test case:
https://jsfiddle.net/lasnochesxl/0qtsvnmd/

But you'll see my problem if you're going on my website cause on jsfiddle, it works ! this is the website where i work : https://www.calais.fr/fr/faq2

As you can see, when you choose another topic, you can see a kind of flash. When i paste my code on jsfiddle and tried it, i understand (i think) that this problem comes from the topic that i closed to open a new one... I searched and i don't 
find the solution. What can i do to solve this problem on firefox / edge and safari (because my program works really well en chrome)

Thanks you.

**Comment by patrickhlauke** [comment]
Not a Boostrap bug, but likely something you've done on your page. changed to a discussion, as issues are for bugs in BS, rather than troubleshooting of specific applications/sites
